data/topic-schema/src/main/java/net/corda/schema/Schemas.java

```diff
@@ -269,5 +269,8 @@ private ScheduledTask() {}
 
         public static final String SCHEDULED_TASK_DB_PROCESSOR = "scheduled.task.db.processor";
         public static final String SCHEDULED_TASK_MAPPER_PROCESSOR = "scheduled.task.mapper.processor";
+        public static final String SCHEDULED_TASK_TOPIC_FLOW_PROCESSOR = "scheduled.task.flow.processor";
```

Comment:
```suggestion
        public static final String SCHEDULED_TASK_FLOW_PROCESSOR_TOPIC = "scheduled.task.flow.processor";
```

I think we should do the same for the 2 above constants for consistency with rest of the cases (I missed that with `SCHEDULED_TASK_DB_PROCESSOR`).

Reply:
Sure. I can add it in this PR if you want?